contrib/kind.sh

```diff
@@ -566,6 +574,21 @@ cleanup_kube_proxy_iptables() {
   done
 }
 
+docker_create_second_interface() {
+  if [ -z "$OVN_EX_GW_NETWORK_INTERFACE" ]; then
+    return
+  fi
```

Comment:
I always prefer to do the if from the main loop

Reply:
so you don't have to go down into the functions to understand what happens

Reply:
Good point! That improves readability a lot

Reply:
Done